alpha/apps/kaltura/lib/myUploadUtils.class.php

```diff
@@ -235,12 +237,25 @@ public static function checkIfFileIsAllowed($uploadFilePath)
 		return in_array($fileType, $fileTypes['allowed']);
 	}
 
-	public static function isFileTypeRestricted($fullPath, $partnerId = null)
+	public static function isFileTypeRestricted($fullPath, $fileName = null, $partnerId = null)
 	{
 		if(!$partnerId)
 		{
 			$partnerId = kCurrentContext::getCurrentPartnerId();
 		}
+		$allowedFileTypes = kConf::get(self::FILE_EXT_WHITELIST, self::SECURITY_MAP, null);
```

Comment:
this also needs to be checked only when FEATURE_FILE_TYPE_RESTRICTION_PERMISSION enabled